conftest.py

```diff
@@ -0,0 +1,26 @@
+# this contains imports plugins that configure py.test for astropy tests.
```

Comment:
What's the story with this added contest.py file? This is pretty cryptic in parts, and I'd like to understand whether this is needed for some of the tests of synphot, whether it provides other benefits, and/or whether this causes any other changes in the behavior of the test system. 

I'd like to request another sentence or two of comments here to clarify what this stuff is for, and/or simplifying if any of this is not strictly needed. 

And also, is the package root directory really the place this file should be? I think in that case it does not get installed into site-packages since it's not within the `poppy` package subdirectory, right? Is that the intended behavior?

Reply:
Story: The way `tox` is being run, `pytest` is not seeing (some of?) your `poppy/conftest.py`, so the test header isn't being displayed correctly in the CI logs (go see for yourself). This fixes the header.

Also see https://github.com/astropy/pytest-astropy-header/issues/15#issuecomment-763247031 .

As for how you want to document this story in a way that you understand, feel free to use the "suggestion" feature. I have been doing this for so long that I might not document it the way that make sense to you.